common/flatpak-context.c

```diff
@@ -707,7 +707,7 @@ unparse_filesystem_flags (const char           *path,
       break;
 
     default:
-      g_warning ("Unexpected filesystem mode %d", mode);
+      g_warning (_("Unexpected filesystem mode %d"), mode);
```

Comment:
This is a "can't happen" message, so I'm not sure it makes sense to burden translators with this.